schemas/dev/parser.schema.json

```diff
@@ -246,13 +246,23 @@
       "type": "object",
       "required": [
         "subject_id",
-        "country_iso3",
-        "admission_date",
         "age",
         "sex_at_birth",
         "ethnicity",
         "pathogen"
       ],
+      "anyOf": [
```

Comment:
looks good!

Reply:
i'm not sure about this, maybe there have been further conversations, but I thought we were keeping admission and enrolment date separate rather than merging into one variable? so if one is blank the other can be used and the users of the data understand what data they are using?

Reply:
This section means that we require one of either enrolment date OR admission date (or both) for the row to be valid against the schema. They're still separate attributes in the data, and you can fill in one, other or both from the data as separate dates.